### PR TITLE
Revisits Adhomai and other sector content

### PR DESCRIPTION
**Pull request by Alberyk**
This pr fixes and updates some old space systems contents.

### CODE REVIEW

File: code/modules/overmap/exoplanets/decor/_areas.dm [hunk 1]
```diff
@@ -22,6 +23,7 @@
 	name = "Raskara Surface"
 	ambience = AMBIENCE_OTHERWORLDLY
 	base_turf = /turf/simulated/floor/exoplanet/barren/raskara
+	area_blurb = "The dark surface of the moon is quite. You feel a strange presence nearby."
```
Comment (Greenjoe12345) [inline code/modules/overmap/exoplanets/decor/_areas.dm:26]:
```suggestion
	area_blurb = "The dark surface of the moon is quiet. You feel a strange presence nearby."
```

**Review by Eyeveri — APPROVED** [review]
The maps and added "code" look good to me.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]
looks good-ish
Gem's requested changes should be implemented though

**Comment by DreamySkrell** [comment]
# nka_merchant.dmm

image
wrong vent types, won't connect to these pipes
area should extend to cover the airlock

image
wrong dir on connector

image
should be the new window spawners

image
windows on engines is old mapping style, exterior should not be visible through engines from the inside 

image
airlocks look correct, both in mapping and code

File: aurorastation.dme [hunk 2]
```diff
@@ -3632,12 +3632,12 @@
 #include "code\ZAS\Zone.dm"
 #include "code\ze_genesis_call\genesis_call.dm"
 #include "interface\interface.dm"
+#include "interface\skin.dmf"
 #include "interface\fonts\fonts_datum.dm"
 #include "interface\fonts\grand_9k.dm"
 #include "interface\fonts\pixellari.dm"
 #include "interface\fonts\spess_font.dm"
 #include "interface\fonts\tiny_unicode.dm"
-#include "interface\skin.dmf"
```
Comment (SleepyGemmy) [inline aurorastation.dme:3640]: don't forget to undo this manually before the PR is merged.

**Pull request merged**